open_spiel/python/examples/benchmark_games.py

```diff
@@ -22,6 +22,8 @@
 from absl import logging
 import pandas as pd
 
+from open_spiel.python import games  # pylint: disable=unused-import
+from open_spiel.python.mfg import games as mfg_games  # pylint: disable=unused-import
```

Comment:
Does this run on any MFG? There seem to be no code to deal with the distribution.

Reply:
Good point. It cannot run on MFG yet.

Reply:
Maybe raise an error or put a TODO if the game is mean field. Or if you are not under time pressure, adding it is a very nice contribution :)

Reply:
Add NotImplementedError